### ISSUE TITLE
Record Tuesday and Thursday Meetings with UX Researcher

### ISSUE THREAD

**Issue by Rabia2219** [hackforla/internship#48]
### Overview
Record Tuesday meetings with UX Researcher(s) for review before Thursday meeting.

### Action Items

- [x] Create folder for recordings
- [ ] Record Tuesday meeting
- [ ] Rename using this format - TuedayNov2nd (example)
- [ ] Add to folder
- [ ] Place link in comment

### Resources/Instructions
[Recording Folder
](https://drive.google.com/drive/folders/1lgc0eeDXvqvqvLOgqsT8lu3EsneeH9VS?usp=sharing)

**Comment by Rabia2219** [comment]
@sniux @priyatalwar @ExperimentsInHonesty 
Requesting that the Tuesday meetings be recorded and placed in [folder](https://drive.google.com/drive/folders/1-c3MtsAUf6Nb8tMgT-ll4CYFzuTyHLc2?usp=sharing). Thanks

**Comment by Rabia2219** [comment]
@sniux 
do you think you can manage this?

**Comment by sniux** [comment]
@Rabia2219 I can record but I don't have the permissions to manage/organize the drive.

**Comment by Rabia2219** [comment]
> @Rabia2219 I did upload the recording for the last time I met Bonnie. I can record but I don't have the permissions to manage/organize the drive.

yes you did :) and that's all that you need to do...just record and upload to the folder linked in the description above...you don't need to manage/organize...thank you :)

**Comment by Rabia2219** [comment]
> @Rabia2219 I did upload the recording for the last time I met Bonnie. I can record but I don't have the permissions to manage/organize the drive.

yes you did and thank you for that 👍 :) that's all you have to keep doing if you are up for it...the folder is linked in the description above...you don't need to manage/organize...just upload the recording to the folder...let me know if that works...thanks

**Comment by sniux** [comment]
@Rabia2219 got it, that works!

**Comment by Rabia2219** [comment]
> @Rabia2219 got it, that works!

Thank you, @sniux...this will help @priyatalwar and I stay on top of your and Tin's assignment...really appreciate it!

**Comment by ExperimentsInHonesty** [comment]
Closing issue, since it is now part of the agenda for the meeting.

**Comment by ExperimentsInHonesty** [comment]
Nevermind.  Not closing this issue.  Because this is a reminder to whoever is assigned to make sure they upload it after the meeting.

**Comment by ExperimentsInHonesty** [comment]
Please sort out these [recordings](https://drive.google.com/drive/folders/1lgc0eeDXvqvqvLOgqsT8lu3EsneeH9VS?usp=sharing) as discussed, making folders that correspond to the dates or content as appropriate.

**Comment by sniux** [comment]
Update for 01-18-2022: sorted all the recordings.

**Comment by ExperimentsInHonesty** [comment]
We no longer need to keep recording meetings.  I am closing this issue
